src/application.ts

```diff
@@ -107,7 +111,8 @@ export default class Application implements Owner {
     if (!mainTemplate) { throw new Error("Could not find main template."); }
 
     let mainLayout = templateFactory(mainTemplate).create(this.env);
```

Comment:
There must be some builder API available or something but I don't know how to get at it from here.